modoboa/admin/management/commands/subcommands/_manage_dkim_keys.py

```diff
@@ -0,0 +1,52 @@
+"""Management command to create DKIM keys."""
+
+from __future__ import print_function, unicode_literals
+
+import os
+
+from django.core.management.base import BaseCommand
+from django.utils.encoding import smart_text
+
+from modoboa.lib import sysutils
+from modoboa.parameters import tools as param_tools
+
+from .... import models
+
+
+class ManageDKIMKeys(BaseCommand):
+    """Command class."""
+
+    def create_new_dkim_key(self, domain):
+        """Create a new DKIM key."""
+        storage_dir = param_tools.get_global_parameter("dkim_keys_storage_dir")
+        pkey_path = os.path.join(storage_dir, "{}.pem".format(domain.name))
+        key_size = (
+            domain.dkim_key_length if domain.dkim_key_length
+            else self.default_key_length)
+        code, output = sysutils.exec_cmd(
+            "openssl genrsa -out {} {}".format(pkey_path, key_size))
```

Comment:
The private key have a chmod 600, the owner must be the user that launch opendkim

Reply:
Does openssl not automatically make the private key chmod 600 when it's generated?

Reply:
Yes, it does. We just need to document that keys will belong to the user running the management command.

Reply:
Yep and check if correct with the installer